_includes/cli.md

```diff
@@ -29,6 +29,15 @@ your client and daemon API versions.
 
 {% endif %}
 
+{% if page.enterprise_only == true %}
+
+> This command is only available on Docker Enterprise Edition.
+>
+> Learn more about [Docker Enterprise products](/ee/supported-platforms/){: target="_blank" class="_"}.
```

Comment:
@thaJeztah  Should we point users to the Enterprise Overview page ( /ee/ ) instead of /ee/supported-platforms/?

Reply:
I was a bit in doubt, because the `/ee/supported-platforms/` is the one that's underneath the "engine" section in the TOC (and also linked from the homepage)

Do you think the other one is better to link to?

Reply:
Yea, there is an overlap between `/ee/supported platforms` and `/ee/`. For Enterprise CLI plugins, I think it makes sense to point to the Overview page underneath Engine. 

I'll approve the PR now. Thanks.